src/lib/components/Segment.svelte

```diff
@@ -114,10 +129,19 @@
       -1px,
       0
     );
-    width: calc((100% - (var(--segments) * var(--padding))) / var(--segments));
-    padding: var(--padding-2x) 0 var(--padding);
+    width: calc(
```

Comment:
This is quite some logic to put in CSS. Do you want to add a comment explaining what this is doing?

Reply:
Good point, I hope w/ the comment it's better. Just tested it adding extra borders to see sizes:
<img width="660" alt="image" src="https://github.com/dfinity/gix-components/assets/98811342/1afffffa-fc6b-4ee2-aabc-d98cd4e3ec05">